### PR TITLE
Issue/65 refactoring

### PR DESCRIPTION
**Pull request by SebastianWagner2**
Contributes to: #65

Refactored the building process of ATDDBrowser

### CODE REVIEW

**Review by kolioOtSofia — APPROVED** [review]

File: packages/AutoTDD-GUI.package/ATDDBrowser.class/class/globalFunctionalityDictionary.st [hunk 1]
```diff
@@ -0,0 +1,11 @@
+constants
+globalFunctionalityDictionary
+	
+	^ OrderedDictionary
+		newFrom: {
+			'Pause All' -> #pauseAll.
+			'Resume All' -> #resumeAll.
+			'Run All' -> #runAll.
+			'Create Overview' -> #openOverview.
+      		'toggle_sound_placeholder' -> #toggleSounds.
```
Comment (benjamin-frost) [inline packages/AutoTDD-GUI.package/ATDDBrowser.class/class/globalFunctionalityDictionary.st:10]: This should probably be something like `Toggle Sound`. Even better would be a solution that shows `Toggle Sound (On/Off)` depending on the current state.

Check out `soundsButtonLabel` and `toggleSounds`; might help

File: packages/AutoTDD-GUI.package/ATDDBrowser.class/instance/buttonHeight.st [hunk 2]
```diff
@@ -1,4 +1,4 @@
-accessing
+building
```
Comment (benjamin-frost) [inline packages/AutoTDD-GUI.package/ATDDBrowser.class/instance/buttonHeight.st]: Sure this is primarily used within building but considering that it returns a constant it should probably be `accessing`. What do you think?

```suggestion
accessing
```

File: packages/AutoTDD-GUI.package/ATDDBrowser.class/instance/heightPerTestingEntity.st [hunk 3]
```diff
@@ -0,0 +1,4 @@
+building
```
Comment (benjamin-frost) [inline packages/AutoTDD-GUI.package/ATDDBrowser.class/instance/heightPerTestingEntity.st]: Same as above

```suggestion
accessing
```

File: packages/AutoTDD-GUI.package/ATDDBrowser.class/instance/testingEntityAmount.st [hunk 4]
```diff
@@ -0,0 +1,4 @@
+building
```
Comment (benjamin-frost) [inline packages/AutoTDD-GUI.package/ATDDBrowser.class/instance/testingEntityAmount.st]: same as above

```suggestion
accessing
```

**Review by benjamin-frost — CHANGES_REQUESTED** [review]
Amazing job 🚀 well done!

File: packages/AutoTDD-GUI.package/ATDDBrowser.class/class/globalFunctionalityDictionary.st (hunk 1 above)
Comment (SebastianWagner2) [reply packages/AutoTDD-GUI.package/ATDDBrowser.class/class/globalFunctionalityDictionary.st:10]: Addressed in #67

**Review by benjamin-frost — APPROVED** [review]

**Pull request merged**